README.md

```diff
@@ -138,7 +135,8 @@ Here a list of all the supported features:
 
 | Feature | Description | Command line |
 | -------- | ----------- | ------------ |
-| `headersToRemove` | List of headers that needs to be ignored for every endpoints. The headers in this list will be dropped and not generated as parameters for the endpoints. | `--featureHeaderToRemove=` |
+| `headersToRemove` | List of headers that needs to be ignored for every endpoints. The headers in this list will be dropped and not generated as parameters for the endpoints. | `-ignoreheaders=` |
+| `supportMoshiReflective` | Boolean value that enables the usage of [`moshi-kotlin`](https://github.com/square/moshi#reflection) into the generated code.<br />By default it is not enabled. | `-supportMoshiReflective` |
```

Comment:
As discussed offline, I'd rather won't have this feature right now. Let's keep the template `moshi-codegen` only. We can still add it if we get a feature request for it.

Reply:
I agree ... I'll keep the diff on a gist such that if we will need to re-add it it won't take very long.

I'll update the branch removing the new feature and references to it